plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeFileOperations.java

```diff
@@ -99,6 +101,35 @@ protected DistributedQueryRunner createQueryRunner()
         }
     }
 
+    @Test
+    public void testCreateTableAsSelect()
+    {
+        assertFileSystemAccesses(
+                "CREATE TABLE test_create_as_select AS SELECT 1 col_name",
+                ImmutableMultiset.<FileOperation>builder()
+                        .addCopies(new FileOperation(STARBURST_EXTENDED_STATS_JSON, "extendeded_stats.json", INPUT_FILE_NEW_STREAM), 1)
+                        .addCopies(new FileOperation(STARBURST_EXTENDED_STATS_JSON, "extendeded_stats.json", INPUT_FILE_EXISTS), 1)
```

Comment:
In accordance with https://github.com/trinodb/trino/pull/19128/commits/80a3ffe74c8e848a0ae5451857334ddd4f6cbec0 , you could use `add(...)` instead of `addCopies(...)`